### ISSUE TITLE
Main docs for UT framework are unmaintained

### ISSUE THREAD

**Issue by diggoran** [Azure/portaldocs#97]
https://github.com/Azure/portaldocs/blob/master/portal-sdk/generated/portalfx-unit-test.md should be the main documentation for the msportalfx-ut framework, but instead https://github.com/Azure/portaldocs/blob/master/portal-sdk/generated/top-extensions-unit-test.md is the most maintained version of the docs. The first link is the first result when Bing searching "msportalfx-ut" and the second link doesn't even make it onto the results list.

**Comment by nickharris** [comment]
@ShrishaKumar we should drop portalFx-unit-test.md from both templates and generated folders in dev/master_template/master branch

**Comment by diggoran** [comment]
I disagree. I don't even know what "top extensions" means, but I do know what "portalfx" means. Why would you favor the one that doesn't even show up in web search results?
